web/src/assets/svg/start-build-button.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="16" height="14" viewBox="0 0 16 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="-4 -4 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
1. if an svg sets a height / width, you can't scale it by setting height/width in css
2. Change the viewbox size to match the viewbox size of the other icons that go in the same place. This makes the icon keep appropximately the same size as it previously had, even though we're now giving it a larger height/width. (not quite, since the viewbox is now 24x24 instead of 16x14, but probably close enough?)